model/fn-execution/src/main/proto/beam_fn_api.proto

```diff
@@ -207,18 +207,13 @@ message BundleApplication {
 // Either an absolute timestamp or a relative timestamp can represent a
 // scheduled execution time.
 message DelayedBundleApplication {
-  // Recommended time at which the application should be scheduled to execute
-  // by the runner. Times in the past may be scheduled to execute immediately.
-  // TODO(BEAM-8536): Migrate usage of absolute time to requested_time_delay.
-  google.protobuf.Timestamp requested_execution_time = 1;
-
   // (Required) The application that should be scheduled.
-  BundleApplication application = 2;
+  BundleApplication application = 1;
```

Comment:
Field indexes in protos should never be changed.

Reply:
Thanks for your comment! I think we should be fine here given:
1.  The usage of `DelayedBundleApplication` is still under development, we never announced an official supported usage so far IIRC.
2. We release versioned SDK, which should also work with versioned runner. I don't think it will break anything.
Would you mind explaining a little bit more about your concern?